lib/cosmos/api/tlm_api.rb

```diff
@@ -246,17 +216,17 @@ def inject_tlm(target_name, packet_name, item_hash = nil, value_type = :CONVERTE
       # TODO: Handle the rest of the parameters
       # inject['send_routers'] = true if send_routers
 
-      Store.instance.get_interfaces.each do |interface|
-        if interface['target_names'].include?(target_name)
-          Store.instance.write_interface(interface['name'], inject)
+      InterfaceModel.all(scope: scope).each do |name, interface|
+        if interface['target_names'].include? target_name
+          Store.write_topic("#{scope}__CMDINTERFACE__#{interface['name']}", inject)
```

Comment:
We already had this mechanism to write to the interface directly so this wasn't too bad.